addOns/selenium/src/main/javahelp/org/zaproxy/zap/extension/selenium/resources/help/contents/intro.html

```diff
@@ -10,8 +10,13 @@ <H1>Selenium</H1>
 	<p>The Selenium add-on provides WebDrivers, for other add-ons, to invoke and remotely
 		control web browsers. It's also bundled the HtmlUnit web browser, an headless Java based
 		web browser.
+		
+	<p>It adds a 'Selenium' script type - these scripts are invoked whenever selenium launches a browser,
```

Comment:
Could link to Selenium JavaDoc/docs (to know about the `WebDriver` interface and other helpful classes, e.g. `WebDriverWait`).